src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss

```diff
@@ -30,12 +30,12 @@ $richie-organization-plugin-title-height: 1.1 * $headings-line-height *
     border: $richie-organization-plugin-border-hover;
     cursor: pointer;
   }
-  &__image {
-    @extend .card-img-top;
-  }
   &__body {
     @extend .card-body;
     @extend .text-center;
+    img {
```

Comment:
maybe `& > img` to pinpoint the image as accurately as possible

Reply:
What do you mean ? I am not sure I understand.

Reply:
I just meant to use a more precise selector. We're not targeting any image somewhere inside the card but the one image directly inside the `__body`